auth0/structure_auth0_connection.go

```diff
@@ -152,6 +152,7 @@ func flattenConnectionOptionsOAuth2(options *management.ConnectionOptionsOAuth2)
 		"scripts":                  options.Scripts,
 		"set_user_root_attributes": options.GetSetUserAttributes(),
 		"non_persistent_attrs":     options.GetNonPersistentAttrs(),
+		"icon_url":                 options.GetLogoURL(),
```

Comment:
We're missing the `go get github.com/auth0/go-auth0@1d904472f214406d98f850fc17191e476f775ff1` at least to ensure this PR passes the tests, afterwards we can make a release and re-update this go.mod to e.g. v0.6.5.